package.json

```diff
@@ -8,7 +8,7 @@
   "license": "The MIT License (MIT)",
   "scripts": {
     "apidocs:build": "cd api-docs && gem install bundler && bundle install && bundle exec middleman build --clean",
-    "apidocs:setup": "git submodule init && git submodule update && cd api-docs && rvm install 2.3.0 && gem install bundle && bundle install",
+    "apidocs:setup": "git submodule init && git submodule update && cd api-docs && gem install bundle && bundle install",
```

Comment:
I removed rvm from command because I am using rbenv. Also, the version called out was old.